docs/docs-app/components/app.js

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
+import React, {Component} from 'react';
 
 import Header from './header';
 
 /* eslint-disable react/display-name*/
-export default ({children}) => (
-  <div className="app">
-    <Header />
-    <div className="app-container">
-      {children}
-    </div>
-  </div>
-);
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      opacity: 1,
```

Comment:
it's weird that you are setting opacity as a state variable given that it has a constant value